src/Symfony/Bridge/Doctrine/ManagerRegistry.php

```diff
@@ -49,7 +49,7 @@ protected function resetService($name)
         }
         $manager->setProxyInitializer(\Closure::bind(
             function (&$wrappedInstance, LazyLoadingInterface $manager) use ($name) {
-                if (isset($this->aliases[$name = strtolower($name)])) {
```

Comment:
this requires a bugfix in 3.3 first instead: we need to use the `normalizedIds` property here instead of lowercasing (internal storage does not use lowercase anymore)